### DISCUSSION TITLE
Turn in Place vs Rotate in Place

### DISCUSSION THREAD

**Discussion by Crazytalkbob** [Sixze/ALS-Refactored#506] [General]
I've been working on implementing some new animations with different foot placement using the Sword Animset Pro asset.

While replacing the animations, I noticed Turn in Place animations are set in AIS_Als_Default, but the Rotate in Place animations are set in AB_Als_Standing -> Standing States. 

I'm curious the reason for the difference, or if one is planned to be replaced with the other eventually. I'm still working out the best way to drop in animations that require new foot placement and will likely try to consolidate these to make it simpler.

**Comment by jomath229** [comment]
Rotate in place is when the character is aiming, and you are not moving and you rotate around "in place". Think of it as this is for when the character isnt moving and the character always faces the direction the camera is looking at. Turn in place is for change of direction when starting to walk in a different direction when the character rotation doesn't necessarily follows the camera direction.
